### PR TITLE
Move over tabs component; split out the tabs list into its own molecule.

### PR DESCRIPTION
**Pull request by lal65**

### CODE REVIEW

File: packages/tabs-list/src/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   *
+   * @param {KeyboardEvent} event - the keydown event
+   */
+  function onKeyDown(event) {
+    const target = event.target;
+    const tabs_list = target.closest('.tabs-list');
+    const key = event.key;
+    const orientation = getOrientation(tabs_list);
+    const active_item = tabs_list.querySelector('.tabs-list__button[aria-selected="true"]');
+console.log(orientation);
```
Comment (lal65) [inline packages/tabs-list/src/js/scripts.js:82]: nuke this

File: packages/tabs-list/src/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            next.click();
+          }
+          break;
+        case 'Home':
+          const first = tabs_list.querySelector('.tabs-list__button:first-child');
+          first.click();
+          break;
+        case 'End':
+          const last = tabs_list.querySelector('.tabs-list__button:last-child');
+          last.click();
```
Comment (lal65) [inline packages/tabs-list/src/js/scripts.js:112]: break;

File: packages/tabs-list/src/js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      }
+    }
+  }
+
+  cms.attach('tabs', context => {
```
Comment (lal65) [inline packages/tabs-list/src/js/scripts.js:117]: Bug!  `tabsList`

File: packages/tabs-list/src/js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    const tabs_lists = context.querySelectorAll('.tabs-list');
+    tabs_lists.forEach(tabs_list => {
+      tabs_list.addEventListener('keydown', onKeyDown);
+      const buttons = tabs_list.querySelectorAll('.tabs-list__button');
+      buttons.forEach(button => {
+        button.addEventListener('click', onClick);
+      });
+    });
+  });
+
```
Comment (lal65) [inline packages/tabs-list/src/js/scripts.js:127]: Add this back in:

```
      // Auto-select a tab if the user is heading there.
      if (context === document) {
        const id = window.location.hash.replace('#', '');
        const button = document.querySelector('[aria-controls="' + id + '"]');
        if (button && button.getAttribute('aria-selected') === 'false') {
          button.click();
        }
      }
```

File: packages/tabs-list/tabs-list.twig [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+{% endif %}
+<div class="{{ classes|join(' ') }}" role="tablist" data-flex-wrap-aware>
+  {% for tab in tabs %}
+    <button class="tabs-list__button"
+            type="button"
+            role="tab"
+            aria-selected="{{ loop.first ? 'true' : 'false' }}"
+            aria-controls="{{ tab.id }}"
+            tabindex="{{ loop.first ? '0' : '-1' }}"
+            id="{{ (region.id ~ '-button') }}">
```
Comment (lal65) [inline packages/tabs-list/tabs-list.twig:18]: tab.id

File: packages/tabs/package.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@psu-ooe/tabs",
+  "version": "1.0.0",
+  "description": "Provides an accessible tabs implementation.",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "dependencies": {
+    "@psu-ooe/base": "^1.0"
```
Comment (lal65) [inline packages/tabs/package.json:10]: Add `@psu-ooe/tabs-list` as dependency.

File: packages/tabs/src/scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    }
+  }
+}
+
+// @TODO: Delete when safari polyfills are no longer needed. */
+// These styles are repeated from the canonical :focus-visible selector.
+.tabs__button.focus-visible {
+  outline: rem(.2) solid $sky-blue;
+  box-shadow: 0 rem(.3) 0 0 $beaver-blue;
+}
```
Comment (lal65) [inline packages/tabs/src/scss/styles.scss:65]: Move to the tabs-list component

**Review by kjl16 — APPROVED** [review]
Looks good - thanks for the updates.

**Pull request merged**